django_mongodb_backend/management/commands/get_encrypted_fields_map.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import DEFAULT_DB_ALIAS, connections
+
+
+class Command(BaseCommand):
+    help = "Generate an encryptedFieldsMap for MongoDB automatic encryption"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--database",
+            default=DEFAULT_DB_ALIAS,
+            help="Specify the database to use for generating the encrypted"
+            "fields map. Defaults to the 'default' database.",
+        )
+
+    def handle(self, *args, **options):
+        db = options["database"]
+        connection = connections[db]
+
+        schema_map = self.generate_encrypted_fields_schema_map(connection)
+
+        self.stdout.write(json.dumps(schema_map, indent=2))
+
+    def generate_encrypted_fields_schema_map(self, conn):
+        schema_map = {}
+
+        for model in apps.get_models():
```

Comment:
This should only include migratable models for given database. See `get_migratable_models` in `django/core/management/commands/migrate.py`.